packages/@react-spectrum/searchwithin/src/SearchWithin.tsx

```diff
@@ -16,23 +16,25 @@ import {FocusableRef} from '@react-types/shared';
 import React, {useCallback, useLayoutEffect, useRef, useState} from 'react';
 import {SpectrumSearchWithinProps} from '@react-types/searchwithin';
 import styles from '@adobe/spectrum-css-temp/components/searchwithin/vars.css';
+import {useField} from '@react-aria/label';
 import {useFormProps} from '@react-spectrum/form';
-import {useLabel} from '@react-aria/label';
+import {useId, useLabels} from '@react-aria/utils';
 import {useProvider, useProviderProps} from '@react-spectrum/provider';
 
 function SearchWithin(props: SpectrumSearchWithinProps, ref: FocusableRef<HTMLElement>) {
   props = useProviderProps(props);
   props = useFormProps(props);
-  let {styleProps} = useStyleProps(props);
-  let {labelProps, fieldProps} = useLabel(props);
   let {
     children,
     isDisabled,
     isRequired,
-    label,
-    'aria-label': ariaLabel,
-    'aria-labelledby': ariaLabelledby
+    label
   } = props;
+  let {styleProps} = useStyleProps(props);
+  let labels = useLabels(props, 'Search');
```

Comment:
Is or should 'Search' be an i18n value?

Reply:
Same as above, I'm not sure we have any at the moment, but it looks like we need them.